model/train_grnet.py

```diff
@@ -212,16 +227,16 @@ def train(model_comp, model_clas, train_dataloader, val_dataloader,
                     model_clas.train()
 
 
-                tb.add_scalar("Val_Loss", val_loss, epoch)
+                tb.add_scalar("Val_Loss", val_loss/(batch_val["incomplete_view"].shape[0] * len(val_dataloader)), epoch)
 
 
 
         #Path(f'/ckpts').mkdir(exist_ok=True, parents=True)
-        batch_loss = batch_loss/len(train_dataloader)
+        val_loss /= len(val_dataloader)
```

Comment:
@DL-Eyes you said you divided by the number of samples right? this is dividing by the number of batches. That's why when I created batch_loss I gave it that name